server/src/main/java/org/elasticsearch/search/vectors/KnnScoreDocQueryBuilder.java

```diff
@@ -73,6 +105,12 @@ protected void doXContent(XContentBuilder builder, Params params) throws IOExcep
             builder.startObject().field("doc", scoreDoc.doc).field("score", scoreDoc.score).endObject();
         }
         builder.endArray();
+        if (fieldName != null) {
+            builder.field("field", fieldName);
+        }
+        if (queryVector != null) {
```

Comment:
Is it possible to have queryVector to be equal to null? When would this happen?

Reply:
Serialization in from an old node